golang/main.go

```diff
@@ -1,7 +1,33 @@
 package main
 
-import "fmt"
+import (
+	"bufio"
+	"fmt"
+	"os"
+
+	"isuct.ru/informatics2022/internal"
+)
+
+func DefaultConsts() internal.FormulaConsts {
+	return internal.FormulaConsts{A: 2.0, B: 3.0}
+}
+
+func DefaultDeltaInfo() internal.DeltaInfo {
+	return internal.DeltaInfo{XStart: 0.11, XEnd: 0.36, XStep: 0.05}
+}
+
+var Xes = []float64{0.08, 0.26, 0.37, 0.48, 0.56}
 
 func main() {
-	fmt.Println("Hello world")
+	stepResult := internal.CalculateResultsWithStep(DefaultConsts(), DefaultDeltaInfo())
```

Comment:
normal pattern in golang - to use `New` pattern, will talk about it little later, but shortly - New is method that returns preconfigured structure and methods are connected to the structure

Reply:
I have implemented this pattern now but not shure that it's implemented right.